src/cpp/rtps/builtin/discovery/endpoint/EDPServer.hpp

```diff
@@ -126,7 +127,7 @@ class EDPServer : public fastrtps::rtps::EDPSimple
      */
     bool process_and_release_change(
             fastrtps::rtps::CacheChange_t* change,
-            const bool& is_remote_participant = false);
+            bool release_from_reader = false);
```

Comment:
I like to force the users of an API to be explicit:
```suggestion
            bool release_from_reader);
```